snippets/facets.liquid

```diff
@@ -157,55 +169,74 @@
                       <fieldset class="facets-wrap parent-wrap {% if filter_type == 'vertical' %} facets-wrap-vertical{% endif %}">
                         <legend class="visually-hidden">{{ filter.label | escape }}</legend>
                         <ul
-                          class="{% if filter_type != 'vertical' %} facets__list{% endif %} list-unstyled no-js-hidden"
+                          class="{{ visual_layout_class }}{% if filter_type == 'vertical' %} facets__list--vertical{% else %} facets__list{% endif %} list-unstyled no-js-hidden"
                           role="list"
                         >
                           {%- for value in filter.values -%}
+                            {% if value.count == 0 and value.active == false %}
+                              {%- assign is_disabled = true -%}
+                            {% else %}
+                              {%- assign is_disabled = false -%}
+                            {% endif %}
+
                             <li class="list-menu__item facets__item{% if forloop.index > 10 and filter_type == 'vertical' %} show-more-item hidden{% endif %}">
                               <label
                                 for="Filter-{{ filter.param_name | escape }}-{{ forloop.index }}"
-                                class="facet-checkbox{% if value.count == 0 and value.active == false %} facet-checkbox--disabled{% endif %}"
+                                class="facets__label facet-checkbox{% if is_disabled %} facet-checkbox--disabled disabled{% endif %}{% if has_visual_representation %} visual-display-parent visual-display-parent--{{ filter.display_style }}{% endif %}{% if value.active %} active{% endif %}"
                               >
                                 <input
                                   type="checkbox"
+                                  class="{% if has_visual_representation %}visually-hidden{% endif %}"
```

Comment:
Weird that only happens in voiceover. It's as if it creates its own border around the element, which is likely what it's doing. To get around this, I just decided to match the height/width of the container. The actual element would go outside of this focus area because of using outline, but this is supposed to be for non-sighted uses anyways so I can't really think of a better (non-hacky) way.

<img width="307" alt="image" src="https://github.com/Shopify/dawn/assets/1019769/53d50bc4-86dc-49a7-a502-232b86dba748">